### DISCUSSION TITLE
[Feature] BlurHash for NSFW images

### DISCUSSION THREAD

**Discussion by truongnguyen3** [immich-app/immich#5882] [Feature Request]
### The feature

I use Immich as a photo app to save all my images/videos. Both SFW and NSFW images are treated at the same level of audiovisual limitations. We should have some features to hide/show NSFW images or add blurhash for NSFW images. All images would be scanned and marked as NSFW images but I suggest we also could mark it manually.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #2451
